ix-dev/community/glances/ix_values.yaml

```
@@ -0,0 +1,8 @@
+images:
+  image:
+    repository: nicolargo/glances
+    tag: "4.3.0.7"
+
+consts:
+  container_name: glances
+  container_port: 61208
```

```suggestion
consts:
  glances_container_name: glances
  internal_server_port: 61208
```

Lets keep formatting as the rest of the apps, so if more containers added or ports. it wont mud the waters! (remember to update docker-compose)!

---

Also done :)